pkg/bgpv1/agent/controller.go

```diff
@@ -176,6 +196,18 @@ func (c *Controller) Start(startCtx hive.HookContext) error {
 		return store.List(), nil
 	})
 
+	secrets, err := c.SecretResource.Store(startCtx)
+	if err != nil {
+		return fmt.Errorf("SecretResource.Store(): %w", err)
```

Comment:
Could we provide an error message explaining what the issue is rather than just the function name?

Reply:
I'm doing the same as the error above it; I think this would only happen in the case of an internal error and I can't really think of a way to explain that, so just copying the error seems reasonable.